employee-accounts-web-app/src/pages/add.jsx

```diff
@@ -1,10 +1,123 @@
-import { Outlet} from "react-router-dom"
+
+import Row from 'react-bootstrap/Row';
+import Col from 'react-bootstrap/Col';
+import Form from 'react-bootstrap/Form';
+import Button from 'react-bootstrap/Button';
+import { useState } from "react";
+import Alert from 'react-bootstrap/Alert';
 
 export function Add() {
-    return(
+  const [validated, setValidated] = useState(false);
```

Comment:
You also need to be setting the info to a list and then displaying it below.

Great job on the validation. Not sure you want to make all fields required?